tests/model/test_scvi.py

```diff
@@ -1256,6 +1210,23 @@ def test_scvi_inference_custom_dataloader(n_latent: int = 5):
     _ = model.get_latent_representation(dataloader=dataloader)
 
 
+def test_scvi_train_custom_dataloader(n_latent: int = 5):
+    # ORI this function could help get started.
+    adata = synthetic_iid()
+    SCVI.setup_anndata(adata, batch_key="batch")
+
+    model = SCVI(adata, n_latent=n_latent)
+    model.train(max_epochs=1)
+    dataloader = model._make_data_loader(adata)
```

Comment:
Does model._make_data_loader exist for all models? We should then add the test to the other models as well?

Reply:
Is the dataloader sufficient to also setup the model and does setup_datamodule work for it?